components/monitoring/grafana/README.md

```diff
@@ -18,50 +18,23 @@ First, create the `appstudio-grafana` namespace on each Prometheus or Grafana cl
 $ oc create namespace appstudio-grafana
 ```
 
-### Grafana Datasources
+### Grafana Datasource
 
-Grafana datasources contain the connection settings to the Prometheus instances. These datasources are stored in secrets in the `appstudio-workload-monitoring` namespace of the **Grafana cluster**.
+Grafana datasources contain the connection settings to the Prometheus instances.
 
-The Prometheus endpoints called by Grafana are protected by an OAuth proxy running as a sidecar container and which checks that the incoming requests contain a valid token. A token is valid if it belongs to a service account of the **Prometheus cluster** which has the RBAC permission to "get namespaces". Such a permission can be obtained with the `cluster-monitoring-view` cluster role.
+A single [kustomization file](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/monitoring/grafana/base/kustomization.yaml#L14)
 
-In a multi-cluster setup, Grafana will have a datasource secret for each instance of Prometheus. 
-A datasource has a name (`DATASOURCE_NAME`), an URL (`PROMETHEUS_URL`) and a token (`GRAFANA_OAUTH_TOKEN`) obtained as follow:
+is defined and 
+it lets us query metrics from the Platform and User Workload Monitoring prom.
 
-`DATASOURCE_NAME` is the name of the datasource as it will appear in Grafana. It is also the name of the secret which will contain the YAML file defining the datasource itself.
-`DATASOURCE_NAME` is an arbitrary name, for example `cluster-1-prometheus-openshift-ds` for Prometheus running in the `openshift-monitoring` namespace of Cluster-1.
+**Remove** any definition of `datasource` you may have when creating or modifying a dashboard from the dashboard json, to use the predefined datasource, for example:
 
-`PROMETHEUS_URL` is obtained from the route created for Prometheus in the `openshift-monitoring` and `appstudio-workload-monitoring` namespaces in the **Prometheus cluster**:
-
-```
-$ export PROMETHEUS_URL_OPENSHIFT=`oc get route/prometheus-k8s -n openshift-monitoring -o json | jq -r '.status.ingress[0].host'`
-
-$ export PROMETHEUS_URL_APPSTUDIO=`oc get route/prometheus-oauth -n appstudio-workload-monitoring -o json | jq -r '.status.ingress[0].host'`
-```
-
-`GRAFANA_OAUTH_TOKEN` is obtained by requesting a token for the `grafana-oauth` service account in the **Prometheus cluster**:
-```
-$ export GRAFANA_SECRET_NAME=$(oc -n appstudio-workload-monitoring get sa/grafana-oauth -o jsonpath="{.secrets[0].name}")
-
-$ export GRAFANA_OAUTH_TOKEN=`oc -n appstudio-workload-monitoring create token grafana-oauth --bound-object-kind Secret --bound-object-name $GRAFANA_SECRET_NAME --duration=8760h`
+``` 
+"datasource": {
+    "type": "prometheus",
+    "uid": "PF224BEF3374A25F8"
```

Comment:
I added a section to explain about the datasource
We have a single datasource and it lets us query metrics from the Platform and User Workload Monitoring Prometheus.
Each team is responsible for it's dashboard.

Reply:
Are you using `templating` with `"regex": ".*-(appstudio)-.*"`?

Reply:
I will rename the datasource to `appstudio-datasource` in another PR, so it will be caught by `".*-(appstudio)-.*"` regex.  
I will update the documentation accordingly in this PR

Reply:
I updated the documentation and added `templating` instructions.
I opened https://issues.redhat.com/browse/RHTAPWATCH-242 to rename the `datasource` to `appstudio-datasource` so it will be caught by the regex.